docs/control_panel_integration/README.md

```diff
@@ -481,6 +484,43 @@ To allow WebShield to handle non-SNI requests properly, include an `ip` field in
 </div>
 
 WebShield will use this data to decide which certificate to serve if a request without Server Name Indication (SNI) arrives. If there are several domains with the specified IPs, WebShield will use the first one alphabetically.
+</details>
+
+#### How to test SSL configuration
+Administrators should see a warning in Settings in UI if no certificates are added:
+WebShield SSL-Cache is not configured. Although, even if a certificate is added, it doesn’t guarantee that the website is working correctly. The certificate may be outdated, invalid, or not applicable to that domain name.
+
+The worst scenario when SSL certificate is not cached or recognised by the WebShield is that the SSL certificate of the Captcha page redirect will not match the initial site the user was visiting. The WebShield will serve it's default that not likely to match with the domain name, or an outdated certificate and this may not be trusted. Thus SSL certificate waning will appear.
+
+To make sure WebShield can serve the Captcha page smoothly the relevant certificates should be in one of the lines of the cache file it uses:
+<div class="notranslate">
+	
+```
+/var/cache/imunify360-webshield/ssl.cache
```

Comment:
Is this an implementation detail (that may change with no warning from devs) or a public interface?

Reply:
This is implementation detail, interface is provided with a documented CLI. The CLI at the moment doesn't provide additional attributes of the cached certificates. Will update this part.